insolar/genesisrefs/refs.go

```diff
@@ -17,52 +17,103 @@
+	ContractRootMember = GenesisRef(insolar.GenesisNameRootMember)
 	// ContractRootWallet is the root wallet contract reference.
-	ContractRootWallet = rootdomain.GenesisRef(insolar.GenesisNameRootWallet)
+	ContractRootWallet = GenesisRef(insolar.GenesisNameRootWallet)
 	// ContractMigrationAdminMember is the migration admin member contract reference.
-	ContractMigrationAdminMember = rootdomain.GenesisRef(insolar.GenesisNameMigrationAdminMember)
+	ContractMigrationAdminMember = GenesisRef(insolar.GenesisNameMigrationAdminMember)
 	// ContractMigrationWallet is the migration wallet contract reference.
-	ContractMigrationWallet = rootdomain.GenesisRef(insolar.GenesisNameMigrationWallet)
+	ContractMigrationWallet = GenesisRef(insolar.GenesisNameMigrationWallet)
 	// ContractDeposit is the deposit contract reference.
-	ContractDeposit = rootdomain.GenesisRef(insolar.GenesisNameDeposit)
+	ContractDeposit = GenesisRef(insolar.GenesisNameDeposit)
+	// ContractStandardTariff is the tariff contract reference.
+	ContractStandardTariff = GenesisRef(insolar.GenesisNameStandardTariff)
 	// ContractCostCenter is the cost center contract reference.
-	ContractCostCenter = rootdomain.GenesisRef(insolar.GenesisNameCostCenter)
+	ContractCostCenter = GenesisRef(insolar.GenesisNameCostCenter)
 	// ContractFeeWallet is the commission wallet contract reference.
-	ContractFeeWallet = rootdomain.GenesisRef(insolar.GenesisNameFeeWallet)
+	ContractFeeWallet = GenesisRef(insolar.GenesisNameFeeWallet)
 
 	// ContractMigrationDaemonMembers is the migration daemon members contracts references.
 	ContractMigrationDaemonMembers = func() (result [insolar.GenesisAmountMigrationDaemonMembers]insolar.Reference) {
 		for i, name := range insolar.GenesisNameMigrationDaemonMembers {
-			result[i] = rootdomain.GenesisRef(name)
+			result[i] = GenesisRef(name)
 		}
 		return
 	}()
 
 	// ContractPublicKeyShards is the public key shards contracts references.
 	ContractPublicKeyShards = func() (result [insolar.GenesisAmountPublicKeyShards]insolar.Reference) {
 		for i, name := range insolar.GenesisNamePublicKeyShards {
-			result[i] = rootdomain.GenesisRef(name)
+			result[i] = GenesisRef(name)
 		}
 		return
 	}()
 	// ContractMigrationAddressShards is the migration address shards contracts references.
 	ContractMigrationAddressShards = func() (result [insolar.GenesisAmountMigrationAddressShards]insolar.Reference) {
 		for i, name := range insolar.GenesisNameMigrationAddressShards {
-			result[i] = rootdomain.GenesisRef(name)
+			result[i] = GenesisRef(name)
 		}
 		return
 	}()
 )
+
+func GenerateTextReference(pulse insolar.PulseNumber, code []byte) *insolar.Reference {
+	hasher := platformpolicy.NewPlatformCryptographyScheme().ReferenceHasher()
+	codeHash := hasher.Hash(code)
+	return insolar.NewReference(*insolar.NewID(pulse, codeHash))
+}
+func GenerateReference(tp string, name string, version int) *insolar.Reference {
```

Comment:
where is space? please, setup your IDE/Editor to avoid such things.

what is `tp`? use descriptive name